common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -434,6 +435,7 @@ def setUp(self):
         AutoAuthPage(self.browser, username=self.USERNAME, email=self.EMAIL,
                      course_id=self.course_id, staff=False).visit()
 
+    @skip('Disable temporarily to get course bookmarks out')
```

Comment:
It looks like it is testing the previous and next buttons and related events.  I don't see why it would be broken.

I think the only change I made was to move the initial visit call from the end of the setup() method to the top of this test, so I didn't really look at it.  Hopefully it is just a test issue, but did we manually test next/previous to ensure we didn't break anything?